src/pages/iou/request/step/IOURequestStepDescription.js

```diff
@@ -70,7 +69,25 @@ function IOURequestStepDescription({
      * @param {String} value.moneyRequestComment
      */
     const updateComment = (value) => {
-        IOU.setMoneyRequestDescription_temporaryForRefactor(transactionID, value.moneyRequestComment);
+        const newComment = value.moneyRequestComment.trim();
+        // Only update comment if it has changed
+        if (newComment !== lodashGet(transaction, 'comment.comment', '')) {
+            navigateBack();
+            return;
+        }
```

Comment:
I think we should move this condition to be below the edit split condition because transaction here could be either from TRANSACTION_DRAFT or TRANSACTION_DRAFT where in the other condition we are working with SPLIT_TRANSACTION_DRAFT